packages/schemas/alterations/next-1709190131-enhance-dau-data-accuracy.ts

```diff
@@ -0,0 +1,18 @@
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    await pool.query(sql`
+      alter table daily_active_users alter column date set default now();
```

Comment:
Do we need to handle the existing data?

Reply:
We don't need that, the original data and new data will stay in the correct time period when doing staticstics:
```sql
select count(*)
from (
  select ${dailyActiveUsers.userId}
  from ${dailyActiveUsers}
  where ${dailyActiveUsers.tenantId} = ${tenantId}
  and ${dailyActiveUsers.date} >= ${from}
  and ${dailyActiveUsers.date} < ${to}
  group by ${dailyActiveUsers.userId}
  having count(*) >= ${minDays} and count(*) <= ${maxDays}
) as active_users;
```